<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::tools::backup::ConfigureBackupDialog</class>
 <widget class="QDialog" name="tt3::tools::backup::ConfigureBackupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Backup workspace</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tt3-tools-backup.qrc">
    <normaloff>:/tt3-tools-backup/Resources/Images/Misc/BackupLarge.png</normaloff>:/tt3-tools-backup/Resources/Images/Misc/BackupLarge.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>41</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QWidget" name="customWorkspacePanel" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="workspaceTypeLabel">
        <property name="text">
         <string>Workspace type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="workspaceTypeComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="locationLabel">
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="locationLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="browsePushButton">
        <property name="text">
         <string>Browse</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-tools-backup.qrc">
          <normaloff>:/tt3-tools-backup/Resources/Images/Actions/BrowseSmall.png</normaloff>:/tt3-tools-backup/Resources/Images/Actions/BrowseSmall.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="backupToLabel">
     <property name="text">
      <string>Backup to:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="backupToLineEdit"/>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="backupToPushButton">
     <property name="text">
      <string>Browse</string>
     </property>
     <property name="icon">
      <iconset resource="tt3-tools-backup.qrc">
       <normaloff>:/tt3-tools-backup/Resources/Images/Actions/BrowseSmall.png</normaloff>:/tt3-tools-backup/Resources/Images/Actions/BrowseSmall.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QRadioButton" name="backupCurrentWorkspaceRadioButton">
     <property name="text">
      <string>Backup current workspace</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="orLabel">
     <property name="text">
      <string>--- Or ---</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QRadioButton" name="backupCustomWorkspaceRadioButton">
     <property name="text">
      <string>Backup custom workspace</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>backupCurrentWorkspaceRadioButton</tabstop>
  <tabstop>backupCustomWorkspaceRadioButton</tabstop>
  <tabstop>workspaceTypeComboBox</tabstop>
  <tabstop>locationLineEdit</tabstop>
  <tabstop>browsePushButton</tabstop>
  <tabstop>backupToLineEdit</tabstop>
  <tabstop>backupToPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="tt3-tools-backup.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tt3::tools::backup::ConfigureBackupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tt3::tools::backup::ConfigureBackupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backupCurrentWorkspaceRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::tools::backup::ConfigureBackupDialog</receiver>
   <slot>_workspaceSourceRadioButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backupCustomWorkspaceRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::tools::backup::ConfigureBackupDialog</receiver>
   <slot>_workspaceSourceRadioButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>workspaceTypeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>tt3::tools::backup::ConfigureBackupDialog</receiver>
   <slot>_workspaceTypeComboBoxCurrentIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browsePushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::tools::backup::ConfigureBackupDialog</receiver>
   <slot>_browsePushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backupToLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>tt3::tools::backup::ConfigureBackupDialog</receiver>
   <slot>_backupTpLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_workspaceSourceRadioButtonClicked()</slot>
  <slot>_workspaceTypeComboBoxCurrentIndexChanged(int)</slot>
  <slot>_browsePushButtonClicked()</slot>
  <slot>_backupTpLineEditTextChanged(QString)</slot>
 </slots>
</ui>
